fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -32,219 +33,219 @@
+    DNF_SIZE,
+    DNF_SIZE_LIMIT,
     // index-related keys
-    INDEX_NAME("index_name"),
-    INDEX_STATE("index_state"),
-    TARGET_INDEX_NAME("target_index_name"),
-    TARGET_INDEX_STATE("target_index_state"),
-    VALUE_KEY("value_key"),
-    PRIMARY_KEY("primary_key"),
-    VALUE("value"),
+    INDEX_NAME,
+    INDEX_STATE,
+    TARGET_INDEX_NAME,
+    TARGET_INDEX_STATE,
+    VALUE_KEY,
+    PRIMARY_KEY,
+    VALUE,
     INDEX_OPERATION("operation"),
-    INITIAL_PREFIX("initial_prefix"),
-    SECOND_PREFIX("second_prefix"),
-    INDEX_VERSION("index_version"),
-    START_TUPLE("start_tuple"),
-    END_TUPLE("end_tuple"),
-    REAL_END("real_end"),
-    RECORDS_SCANNED("records_scanned"),
-    ORIGINAL_RANGE("original_range"),
-    SPLIT_RANGES("split_ranges"),
-    REASON("reason"),
-    LATEST_ENTRY_TIMESTAMP("latest_entry_timestamp"),
-    EARLIEST_ADDED_START_TIMESTAMP("earliest_added_start_timestamp"),
-    REMOVE("remove"),
-    TEXT_SIZE("text_size"),
-    UNIQUE_TOKENS("unique_tokens"),
-    AVG_TOKEN_SIZE("avg_token_size"),
-    MAX_TOKEN_SIZE("max_token_size"),
-    AVG_POSITIONS("avg_positions"),
-    MAX_POSITIONS("max_positions"),
-    TEXT_KEY_SIZE("text_key_size"),
-    TEXT_VALUE_SIZE("text_value_size"),
-    TEXT_INDEX_SIZE_AMORTIZED("text_index_size_amortized"),
-    WROTE_INDEX("wrote_index"),
-    NEW_STORE("new_store"),
-    RECORDS_WHILE_BUILDING("records_while_building"),
-    DOCUMENT("document"),
-    SESSION_ID("session_id"),
-    INDEXER_SESSION_ID("indexer_session_id"),
-    INDEXER_ID("indexer_id"),
-    INDEX_STATE_PRECONDITION("index_state_precondition"),
-    INITIAL_INDEX_STATE("initial_index_state"),
+    INITIAL_PREFIX,
+    SECOND_PREFIX,
+    INDEX_VERSION,
+    START_TUPLE,
+    END_TUPLE,
+    REAL_END,
+    RECORDS_SCANNED,
+    ORIGINAL_RANGE,
+    SPLIT_RANGES,
+    REASON,
+    LATEST_ENTRY_TIMESTAMP,
+    EARLIEST_ADDED_START_TIMESTAMP,
+    REMOVE,
+    TEXT_SIZE,
+    UNIQUE_TOKENS,
+    AVG_TOKEN_SIZE,
+    MAX_TOKEN_SIZE,
+    AVG_POSITIONS,
+    MAX_POSITIONS,
+    TEXT_KEY_SIZE,
+    TEXT_VALUE_SIZE,
+    TEXT_INDEX_SIZE_AMORTIZED,
+    WROTE_INDEX,
+    NEW_STORE,
+    RECORDS_WHILE_BUILDING,
+    DOCUMENT,
+    SESSION_ID,
+    INDEXER_SESSION_ID,
+    INDEXER_ID,
+    INDEX_STATE_PRECONDITION,
+    INITIAL_INDEX_STATE,
     SHOULD_BUILD_INDEX("do_build_index"),
     SHOULD_CLEAR_EXISTING_DATA("clear_existing_data"),
     SHOULD_MARK_READABLE("should_mark_readable"),
-    RESULT("result"),
-    INDEXER_CURR_RETRY("indexer_curr_retry"),
-    INDEXER_MAX_RETRIES("indexer_max_retries"),
-    DIRECTOY("directory"),
-    SOURCE_INDEX("source_index"),
-    SOURCE_FILE("source_file"),
-    CONTINUED_BUILD("continued_build"),
-    INDEXING_METHOD("indexing_method"),
-    ALLOW_REPAIR("allow_repair"),
-    INDEXING_POLICY("indexing_policy"),
-    INDEXING_POLICY_DESIRED_ACTION("indexing_policy_desired_action"),
+    RESULT,
+    INDEXER_CURR_RETRY,
+    INDEXER_MAX_RETRIES,
+    DIRECTOY,
```

Comment:
It looks like there was a typo in the name
```suggestion
    DIRECTOY("directory"),
```

Reply:
I went ahead and changed `DIRECTOY` to `DIRECTORY` because I'm pretty sure the name was a typo